### PR TITLE
feat: describe example data usage in play

### PR DESCRIPTION
**Pull request by yT0n1**
closes https://github.com/camunda/play/issues/494

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by yT0n1** [comment]
@camunda/tech-writers this is my first docs PR, so I tagged you even though this is a small change. Let me know if you need anything from me/something is missing.

File: docs/components/modeler/web-modeler/play-your-process.md [hunk 1]
```diff
@@ -44,8 +46,13 @@ However, actions in Play cannot be triggered by any external system, such as ext
...
 You have a few options to mock an external system:
 
-- In **Implement** mode, hard-code an example payload in the task or event **Output** section.
+- In **Implement** mode, hard-code an example payload in the task or event's **Example data** section in the properties panel.
 - When completing a task or event, use the secondary action to complete it with variables.
+- When filling forms or setting variables from Play, you can also save the variables to the BPMN file as example data to reuse them in future sessions.
+
+Play automatically uses example data from the BPMN file for many events and task types.
+If you want to use different data, you can override the example data by opening the secondary action menu on an element.
+The new data set will take precedent over the example data from the BPMN file for future Play sessions.
```
Comment (huygur) [inline docs/components/modeler/web-modeler/play-your-process.md:55]: this only happens if we save it though, right? it is possible to override the example data, but not save it, so it is only overridden for the existing play session.

(I might be wrong though, but this is how I remember it worked when testing the feature)

Comment (yT0n1) [reply docs/components/modeler/web-modeler/play-your-process.md:57]: @huygur Not quite, if you set data, this data will always be cached and used for the next session, too.

**Review by huygur — DISMISSED** [review]

**Comment by yT0n1** [comment]
@christinaausley I had another look at the actual data handling page based on some feedback [here](https://camunda.slack.com/archives/C049ZK6J430/p1722498431324179)
Could you have another look and let me know if this is ok to merge?

**Review by christinaausley — APPROVED** [review]

**Pull request merged**